lesson16/index.js

```diff
@@ -0,0 +1,124 @@
+// const url = "https://myjson.dit.upm.es/api/bins/hyv9";
+const url = "https://myjson.dit.upm.es/api/bins/d9vx";
+const newsBlock = document.getElementById("news");
+const newsTabs = document.getElementById("news_Tabs");
+
+const createNewsTabs = (allData) => {
+  allData.forEach((content, index) => {
+    const li = document.createElement("li");
+    li.classList.add("tab_item");
+    li.id = content.category;
+    li.textContent = content.category;
+    newsTabs.appendChild(li);
+    if(content.isActive === true){
+      li.classList.add("is-active");
+    }
+  });
+};
+
+const createNewsContent = (allData) => {
+  // 初期表示用の処理
+  const primaryElement = document.createElement("div");
+  primaryElement.classList.add("news_Wrapper");
+  const lists = document.createElement("ul");
+  lists.classList.add("news_Lists");
+  lists.id = 'news_Lists';
+  const primaryImageWrapper = document.createElement("div");
+  primaryElement.appendChild(lists);
+  primaryElement.appendChild(primaryImageWrapper);
+  newsBlock.appendChild(primaryElement);
+
+  const selectedData = allData.find((data) => data.isActive === true);
```

Comment:
IMO
isActive is a boolean.
```suggestion
  const selectedData = allData.find((data) => data.isActive);
```

Reply:
6efda13

Changed the description of the conditional part.